brianlong/www.validators.app
# 178709440 performance speedometer for asns

#### What's this PR do?
Add speedometer in asn list table

#### How should this be manually tested?
- make sure you have IP's in the db
- run dev/asn_logic_demo.rb
- check if speedometer is present at /data-centers/mainnet?sort_by=asn

#### What are the relevant tickets?
https://www.pivotaltracker.com/story/show/178709440

#### Task completed checklist:
- [Y] Is there appropriate test coverage?
- [Y] Did I take into consideration possible security vulnerabilities?
- [Y] Are the controllers secure?
- [Y] Was every important matter documented?

#### Things to watch out for
- [Y] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
